config/initializers/_config.rb

```diff
@@ -52,5 +52,6 @@ module Convection
     vibrations_url:
       ENV['VIBRATIONS_URL'] || 'https://admin-partners-staging.artsy.net',
     unleash_url: ENV['UNLEASH_URL'],
-    unleash_token: ENV['UNLEASH_TOKEN']
+    unleash_token: ENV['UNLEASH_TOKEN'],
+    send_new_receipt_email: ENV['SEND_NEW_RECEIPT_EMAIL'] || 'false'
```

Comment:
This file is a good place to convert configuration to the necessary type (`true`/`false` in this case, rather than a string). You can borrow from the example above that tests for `== 'true'` so that the code locations that reference this can safely expect a boolean.

Reply:
thank you, this will make it more straightforward to reference it in the code.